webapp/src/components/chip/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Tooltip from 'components/tooltip';
+
+import './styles.scss';
+
+type ChipProps = {
+    text: string
+    extraClass?: string
+}
+
+const Chip = ({text, extraClass = ''}: ChipProps) => (
+    <Tooltip tooltipContent={text}>
+        <div className={`chip Badge__box text-truncate ${extraClass}`}>{text}</div>
```

Comment:
Why `Badge__box` class had different casing than other classes?

Reply:
Yes this is inconsistent naming but it's from Mattermost webapp we have not created this, using directly from MM styling